packages/nextra/styles/steps.css

```
@@ -9,10 +9,12 @@
     @apply x:border-0 x:pb-0 x:font-semibold x:tracking-tight x:mt-8 x:text-2xl;
     /* https://github.com/tailwindlabs/tailwindcss/issues/15597#issuecomment-2582673546 */
     @apply x:before:bg-gray-100 x:dark:before:bg-neutral-800;
```

```suggestion
    @apply x:before:bg-gray-200 x:dark:before:bg-neutral-800;
```
make bg more contrast